rts/Map/ReadMap.cpp

```diff
@@ -568,13 +568,6 @@ void CReadMap::UpdateHeightBounds(int syncFrame)
 		tempHeightBounds.y = std::max(h, tempHeightBounds.y);
 	}
 #else
```

Comment:
The complexity / performance tradeoff is not favorable here. This function now performs an order of magnitude more work for a result that is identical within measurement error to the ST version, and is much longer to read and harder to understand at a glance. Moreover, two obvious optimizations have not been considered: an early-out dirty check if the heightmap was changed at all and use of SSE intrinsics (`minps`, don't ever trust the compiler to emit these for you) to vectorize the loop.

The code can stay as a reference if desired, but please place it within the `#if 0` block.

Reply:
Done. Employed interesting lib for that. The code is more readable than pure SSE intrinsics

Reply:
Nice, but does it sync? What controls which SIMD instruction set(s) this library uses?

Reply:
Should sync. Emitted assembly uses instructions {min,max}{ps,ss}.
Though all recent changes, requested during the review of this PR, are not battle tested.

The level SIMD instructions is defined by conditional compilation https://xsimd.readthedocs.io/en/latest/api/instr_macros.html and compilers are supposed to automatically define definitions like `__SSE__`. At least this is what stackoverflow says. In worst case scenario `XSIMD_X86_INSTR_SET` to appropriate `XSIMD_X86_*_VERSION` manually through CMakeLists.